src/app/components/elements/panels/Banner.tsx

```diff
@@ -7,23 +7,39 @@ export interface BannerProps {
   subtitle: string;
   link: string;
   src: string;
+  secondary?: boolean;
   alt?: string;
 }
 
-export const Banner = ({ id, title, subtitle, link, src, alt, children }: PropsWithChildren<BannerProps>) => {
+export const Banner = ({
+  id,
+  title,
+  subtitle,
+  link,
+  src,
+  secondary,
```

Comment:
Small one, but this variable should not be placed between `src` and `alt`, as those two are image properties, and it makes sense for them to be next to each other (same thing applies to the props type).

Related to the previous, it's not really that clear that `src` and `alt` are image properties, you need to read the code to know that, which means that it's not the best choice of parameter name. The `alt` one is specially confusing now that there is another parameter called `secondary`. `imageSource` and `imageDescription` would be better names, I think.

Finally, you also have the option to replace `secondary` with `primary = true`, but only if you think that feels a bit more natural. There is nothing wrong with the way you've done it.

Reply:
Renamed `src` / `alt` to `imageSource` / `imageDescription` and reordered. 

I also changed it from `secondary` as a boolean, to `color` which is primary or secondary